api/src/main/java/org/xwiki/filemanager/internal/reference/DefaultUniqueDocumentReferenceGenerator.java

```diff
@@ -75,12 +86,31 @@ public class DefaultUniqueDocumentReferenceGenerator
      */
     @Inject
     private ComponentManager componentManager;
+    
+    /**
+     * Transform a name according to the current name strategy, if the configuration is set to use transformation.
+     * Else it will just return the given name.
+     *
+     * Taken from org.xwiki.model.validation.script.ModelValidationScriptService
+     *
+     * @param name the name to transform.
+     * @return the transformed named.
+     */
+    private String transformName(String name)
```

Comment:
For Java we have some templates that you can import into your IDE in order to apply the XWiki codestyle automatically, info can be found at https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/
I noticed some codestyle problems on the java files, applying the templates should fix all of them :)